cypress/integration/home.spec.ts

```diff
@@ -1,4 +1,35 @@
-it('should redirect to home page', () => {
+const getTaskInput = () => cy.get('[data-testid="input-task"]');
+const getAddTaskButton = () => cy.get('[data-testid="add-task-button"]');
+const getTaskInputError = () => getTaskInput().find('.Mui-error');
+const getTaskItem = () => cy.get('.MuiListItem-root');
+
+it('should appear label message error when form is submitted empty', () => {
   cy.visit('/');
-  cy.location('pathname').should('eq', '/');
+  getAddTaskButton().click();
+  getTaskInputError().should('be.visible');
+});
+
+it('should appear label message error when task title length is greater then 20 caracters', () => {
+  cy.visit('/');
```

Comment:
I suggest creating a beforeEach command to avoid that repetion:

```ts
beforeEach(() => {
  cy.visit('/');
});

... your tests here

Reply:
Resolved on 17cbf444959bb406808297bada30d5755b9984c1